apps/examples/araplayer/AudioUtils.h

```diff
@@ -0,0 +1,160 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __AUDIO_UTILS_H__
+#define __AUDIO_UTILS_H__
+
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdarg.h>
+#include <pthread.h>
+#include <unistd.h>
+#include <time.h>
+#include <sys/time.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <media/MediaTypes.h>
+
+#include "AudioCommon.h"
+
+#define CLEAN(x)  \
+	if (x)        \
+		delete x; \
+	x = NULL;
+
+using namespace media;
+
+// MPEG version
+
+#define MPEG_1 3
+#define MPEG_2 2
+#define MPEG_UNDEFINED 1
+#define MPEG_2_5 0
+
+// MPEG layer
+#define LAYER_1 3
+#define LAYER_2 2
+#define LAYER_3 1
+#define LAYER_UNDEFINED 0
+
+// Bitrate index
+#define BITRATE_FREE 0x0 // Variable Bit Rate
+#define BITRATE_BAD 0xf  // Not allow
+
+// Sample Rate index
+#define SAMPLE_RATE_UNDEFINED 0x3
+
+// aau size = frame samples * (1 / sample rate) * bitrate / 8 + padding
+//            = frame samples * bitrate / 8 / sample rate + padding
+// Number of frame samples is constant value as below table:
+//        MPEG1  MPEG2(LSF)  MPEG2.5(LSF)
+// Layer1  384     384         384
+// Layer2  1152    1152        1152
+// Layer3  1152    576         576
+#define MPEG_LAYER1_AAU_SIZE(sr, br, pad) (384 * ((br)*1000) / 8 / (sr) + ((pad)*4))
```

Comment:
please provide space before and after * 
ex: #define MPEG_LAYER1_AAU_SIZE(sr, br, pad) (384 * ((br) * 1000) / 8 / (sr) + ((pad) * 4))